src/com/projectkorra/projectkorra/waterbending/Torrent.java

```diff
@@ -355,7 +348,7 @@ private boolean launch() {
 				}
 
 				final Block block = blockloc.getBlock();
-				if (!doneBlocks.contains(block) && !GeneralMethods.isRegionProtectedFromBuild(this, blockloc)) {
+				if (!doneBlocks.contains(block) && !RegionProtection.isRegionProtected(this, blockloc)) {
 					if (isTransparent(this.player, block)) {
 						this.launchedBlocks.add(new TempBlock(block, Material.WATER));
 						doneBlocks.add(block);

```

Comment:
Not to do with these lines but beneath it there is an if that just returns true or false that can be compressed into returning the if's condition inverted.